### ISSUE TITLE
Cannot import valid OpenApi3 yaml

### ISSUE THREAD

**Issue by gongdo** [a85/POSTMan-Chrome-Extension#7059]
**Describe the bug**
Postman cannot import a valid OpenApi 3 yaml which can be converted to collection by `openapi2postmanv2` cli.

**To Reproduce**
1. open Postman (v7.3.5)
2. Click [Import]
3. Paste follow yaml to [Paste Raw Text]
4. Click [Import]

```
openapi: "3.0.0"

info:
  version: 1.0.0
  title: My API server
  description: |
    My something service.

externalDocs:
  description: Repository
  url: "https://somewhere.over.the.rainbow"

tags:
  - name: content
    description: My contents

paths:
  /:
    get:
      operationId: getCollections
      summary: get available collections
      parameters:
        - $ref: '#/components/parameters/pagingOffsetParam'
        - $ref: '#/components/parameters/pagingLimitParam'
      responses:
        200:
          description: OK
          headers:
            Link:
              description: related links such as `prev`, `next`; see RFC 5988 Web Linking
              schema:
                type: string
          content:
            application/json:
              schema:
                type: array
                items:
                  $ref: '#/components/schemas/Collection'

components:
  parameters:
    pagingOffsetParam:
      in: query
      name: offset
      required: false
      schema:
        type: integer
        minimum: 0
        default: 0
      description: The number of items to skip, starting from 0.
    pagingLimitParam:
      in: query
      name: limit
      required: false
      schema:
        type: integer
        minimum: 1
        maximum: 50
        default: 15
      description: The maximum number of items to return from the offset.

  schemas:
    Collection:
      type: object
      description: A collection of cards.
      properties:
        id:
          type: string
          description: unique id
        createdTime:
          type: string
          format: date-time
          description: created time in RFC-3339 include milliseconds
        modifiedTime:
          type: string
          format: date-time
          description: modified time in RFC-3339 include milliseconds
        title:
          type: string
          description: A name of the collection.
```

**Expected behavior**
Postman imports the openapi yaml to a new collection.

**Screenshots**
![image](https://user-images.githubusercontent.com/5451328/62433048-cb33d200-b76d-11e9-8d95-c8c7a5a1e679.png)

There's no further error message or log.

**App information (please complete the following information):**
 - Native App
 - Postman Version 7.3.5
 - OS: Windows 10 1903 / MAC OS 10.14.5 Mojave

**Additional context**
I wonder why it couldn't import this valid openapi yaml since `openapi2postmanv2` tool works fine.
Does postman use different way to import openapi? Follow command works well.
```
$ openapi2postmanv2 -s api.yaml -o collection.json -p
```
It would be better if I could see detail error message about it.

**Comment by Isengo1989** [comment]
Same here with the snap postman app on Ubuntu 18.04 and a bigger YAML/JSON Open API 3 File

https://ufile.io/ms23aix5

**Comment by Dhroov7** [comment]
@gongdo Thanks for reporting the issue.
Can you check this again in 7.6.0?
I think it's importing now.

**Comment by gongdo** [comment]
@Dhroov7 I'm using 7.6.0 and my complex OpenAPI yaml imported as expected!

**Comment by DannyDainton** [comment]
Closing this issue as it has now been resolved in version: 7.6.0
Please feel free to reopen with the steps to reproduce if the issue persists.
